### DISCUSSION TITLE
Support for more than one level of threading for discussion comments

### DISCUSSION THREAD

**Discussion by dmsimard** [community/community#3200] [Discussions]
Hi,

Discussions are currently threaded like the following example:
```
> tree discussion/
discussion/
├── comment-1
│   ├── reply-1
│   ├── reply-2
│   ├── reply-3
│   ├── reply-4
│   └── reply-5
└── comment-2
    ├── reply-1
    ├── reply-2
    ├── reply-3
    ├── reply-4
    └── reply-5
```

The replies are shown in chronological order even though reply-5 might be replying to reply-2, mixed up in the other replies.

In many cases, replies are in fact replies to other replies, better represented as such:
```
> tree discussion/
discussion/
├── comment-1
│   ├── reply-1
│   │   ├── reply-3
│   │   └── reply-4
│   └── reply-2
│       └── reply-5
└── comment-2
    ├── reply-1
    ├── reply-2
    └── reply-3
        └── reply-4
            └── reply-5
```

The current layout makes it hard to follow who is replying to who/what (even when using quotes and formatting) and this is compounded by the fact that threads are eventually collapsed and must be expanded by clicking on ``View more``.

It would be a great improvement to have support for multiple levels of threading like other discussion platforms such as Reddit.

Thanks !

**Comment by evi-liu** [comment]
Thanks for the feedback! We will be gauging customer impact via upvotes to inform prioritization.

**Reply by dmsimard** [reply]
For me this is less about prioritization and more about threading but thanks for looking into that :)

Would you consider implementing multiple levels of threading like I mentioned ? It would really help to understand and better frame replies to a comment in context.

**Reply by adam-debkowski** [reply]
Hello, is there any news regarding this feature? @evi-liu
